src/js/components/conversation.jsx

```diff
@@ -62,7 +62,32 @@ export class ConversationComponent extends Component {
     }
 
     render() {
-        const messages = this.props.conversation.messages.map((message) => {
+        const processedMessages = [...this.props.conversation.messages];
+        let lastAuthor;
+        processedMessages.forEach((message, index) => {
+            const author = message.role === 'appUser' ? message.role : `${message.role}_${message.name}`;
```

Comment:
@mspensieri this logic is there to make sure messages from different appMakers won't be grouped together. What if I only use `message.name` when looking at appMakers? That way, messages from different appMakers will be separated, but whispers and messages from same appMaker will show up together.